src/main/java/uk/gov/digital/ho/hocs/domain/repositories/MessageLogRepository.java

```diff
@@ -45,4 +47,7 @@ void updateProcessedTime(@Param("messageId") String messageId,
 
     Stream<MessageLog> findByStatusAndReceivedBefore(Status status, LocalDateTime to);
 
+    @Cacheable(value="messageLogcountByStatusIn")
```

Comment:
Should this be `Cacheable`? The job runs every 10/15 minutes so the 30m expiry isn't appropriate. Happy if I've missed something here?